src/server/game/Scripting/ScriptMgr.h

```diff
@@ -2333,6 +2337,7 @@ class ScriptMgr
     void OnAfterInitializeLockedDungeons(Player* player);
     void OnAfterUpdateEncounterState(Map* map, EncounterCreditType type, uint32 creditEntry, Unit* source, Difficulty difficulty_fixed, DungeonEncounterList const* encounters, uint32 dungeonCompleted, bool updated);
     void OnBeforeWorldObjectSetPhaseMask(WorldObject const* worldObject, uint32& oldPhaseMask, uint32& newPhaseMask, bool& useCombinedPhases, bool& update);
+    bool OnIsAffectedBySpellModCheck(SpellInfo const* affectSpell, SpellInfo const* checkSpell, SpellModifier const* mod);
```

Comment:
misc take, but mod -> spellMod?

Reply:
It reflects the function it used at: https://github.com/azerothcore/azerothcore-wotlk/pull/9903/files#diff-b85d08735de1a07454ea9d244c37ce7ac6ece09993b475d6ba455237606488cfL1275